doc/src/content/xdocs/gettingstartedpython.xml

```diff
@@ -73,9 +80,8 @@ $ python
       </p>
       <source>
 $ cd lang/py/
-$ python setup.py install
+$ pip install -e .
```

Comment:
I suggest we write this `python3 -m pip install -e .`

Reply:
Good point, thanks